src/yang/example-compb-diffserv-filter-policy.yang

```diff
@@ -131,108 +94,84 @@ module example-compb-diffserv-filter-policy {
           "/policy:classifier" +
           "/policy:inline/policy:filter" +
           "/diffserv:filter-params" {
-    case forwarding-class {
-        uses forwarding-class;
-      description
-        "Filter Type Internal-loss-priority";
-    }
     case internal-loss-priority {
         uses loss-priority;
       description
         "Filter Type Internal-loss-priority";
     }
     description 
       "Augments Diffserv Classifier with vendor" +
-      " specific types";
+      " specific types.";
```

Comment:
s/" specific types/"specific types/

Reply:
in context of previous line, a space is needed